odkx-src/sync-endpoint-manual-setup.rst

```diff
@@ -99,6 +99,7 @@ Setup instructions:
     .. note::
 
       It is important that the right side of the colon stays as 8080. This is the internal port that the web server is looking for.
```

Comment:
@sanyud24 excellent work. One thing to point out is that people might need to change from 8080 if another process on their computer is using it. 

So the message would be improved by saying "as 8080 or whatever port you are using (8080 is the default)."

I would also add a reminder at the end to say only "Reminder that only one process can own a port at a time so if another process on the computer is using port 8080 there will be a conflict and sync-endpoint may not function correctly."